Source/Tests/LinuxSyscalls/SignalDelegator.cpp

```diff
@@ -456,9 +456,12 @@ namespace FEX::HLE {
   }
 
   uint64_t SignalDelegator::GuestSigProcMask(int how, const uint64_t *set, uint64_t *oldset) {
-    if (!!oldset) {
-      *oldset = ThreadData.CurrentSignalMask.Val;
-    }
+    // Ordering matters in setting plus getting
```

Comment:
These two lines are difficult to parse - any chance there's a clearer way of putting it?

Reply:
Updated comment.